verifyFiles.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+#Script to check whether the output files generated by HCP pipeline scripts are the same or not. It will check for the match between two files with respect the filename, size, content, modification time, distance etc.
+#Maintainters : Big Data Lab Team, Concordia University.
+#email : tristan.glatard@concordia.ca, laletscaria@yahoo.co.in
+
+import os
+import sys
+import subprocess
+import argparse
+import hashlib
+from collections import defaultdict,OrderedDict
+#study_folder_details_dict_list is a list for storing ordered dictionaries containing the details regarding the files of  individual subjects.
+study_folder_details_dict_list=[]
+#Method list_files_and_dirs is used for listing files and directories present in the input directory.
+#Input Argument : Directory Path
+def list_files_and_dirs(dirPath):
+        listFileAndDirs=[]
+	rootDir=dirPath
+        for dir_, _, files in os.walk(rootDir):
+           listFileAndDirs.append(os.path.relpath(dir_, rootDir))
+           for fileName in files:
+              relDir=os.path.relpath(dir_, rootDir)
+              relFile=os.path.join(relDir, fileName)
```

Comment:
Filenames starts with the subject folder as the root this way " 100307/T1w" . So keeping them as it is.